README.md

```diff
@@ -68,6 +82,8 @@ docker run \
   -vv run "united states" "santa rosa" "new mexico" 3570670
 ```
 
+-v bna-data:/usr/src/app/data \
```

Comment:
I think this may be a typo here and can be removed
```suggestion

```

Reply:
Ah yes, good catch! That was a remnant of my experimentation.